### ISSUE TITLE
Reverse proxy

### ISSUE THREAD

**Issue by RogueEmailing** [Sigma-Production/ptero-eggs#77]
So I'm trying to create a wordpress site and use the reverse proxy from another system on the network (using nginx proxy manager) but it wont allow me to use ssl. or at least when i do it decides that it doesnt want to load the styles and in the console it says that it was sent over HTTPS, well you can see the image. What can i do to resolve this?
![image](https://github.com/Sigma-Production/ptero-eggs/assets/170282299/15ee20b4-3c5e-4b31-991e-03f22bdac517)

**Comment by RogueEmailing** [comment]
This is what the site looks like when forcing SSL using nginx proxy manager
![image](https://github.com/Sigma-Production/ptero-eggs/assets/170282299/a5cd47a1-d74e-4ec8-b8a7-2fe321baa9a3)

**Comment by Gvolexe** [comment]
Same here did you fix it?

**Comment by finnie2006** [comment]
> So I'm trying to create a wordpress site and use the reverse proxy from another system on the network (using nginx proxy manager) but it wont allow me to use ssl. or at least when i do it decides that it doesnt want to load the styles and in the console it says that it was sent over HTTPS, well you can see the image. What can i do to resolve this? ![image](https://private-user-images.githubusercontent.com/170282299/342275929-15ee20b4-3c5e-4b31-991e-03f22bdac517.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3OTA2NzUsIm5iZiI6MTcyOTc5MDM3NSwicGF0aCI6Ii8xNzAyODIyOTkvMzQyMjc1OTI5LTE1ZWUyMGI0LTNjNWUtNGIzMS05OTFlLTAzZjIyYmRhYzUxNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNFQxNzE5MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YWI5MGZmOWI5YjJmY2FmYzg2YWMwNDMzMzkxNjJjZTIzYzQ2MjI4MmFmZTkzOWFlZmVhNjc1NDYyOTFjYTAwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.y8pakeXh2PMG7q_B2upxDIXa4sHYbOpNz48YIo45N3U)

Try changing the url to https in the wordpress settings after installation, check if it works

**Comment by pasmax2** [comment]
have you solved the problem @StellarFr0st or@Gvolexe

**Comment by Gvolexe** [comment]
@pasmax2 yes i did you need to reinstall the wordpress site and start the instalation through the proxy (https://example.com)
And not the ip

**Comment by pasmax2** [comment]
after checking these wordpress which does not send in forced https

![Image](https://github.com/user-attachments/assets/dceb14b6-8924-4ffa-a54d-59e945307a36)
